doc/source/ray-core/tasks.rst

```diff
@@ -14,7 +14,7 @@ Ray enables arbitrary functions to be executed asynchronously on separate Python
             :start-after: __tasks_start__
             :end-before: __tasks_end__
 
-        See the `ray.remote package reference <package-ref.html>`__ page for specific documentation on how to use ``ray.remote``.
+        See the `ray.remote package reference <ray.remote.html>`__ page for specific documentation on how to use ``ray.remote``.
```

Comment:
I think it needs to be `./api/doc/ray.remote.html`

this package is rendered at `/en/master/ray-core/tasks.html`, and the target is at `/en/master/ray-core/api/doc/ray.remote.html`

Reply:
there are also many other references to `ray-core/package-ref.html` in the code base.

maybe the right thing is to add back `package-ref.html` with a redirect.

@angelinalg @simran-2797 do you know how `package-ref.html` gets lost?

Reply:
Thanks for investigatin, @aslonnie. The `package-ref.html` reference was a mystery to me. I wonder if @peytondmurray understands it.